libraries-bom-table-generation/helpers/handwritten_libraries_javadocs_modules.txt

```diff
@@ -0,0 +1,14 @@
+googleapis/java-bigquery java-bigquery v2.29.0 google-cloud-bigquery
```

Comment:
This seems like an intermediary file. Do we need to store it in git?

Reply:
removed with refactor